contracts/crowdsale/ZxcCrowdsale.sol

```diff
@@ -266,29 +301,22 @@ contract ZxcCrowdsale is
   /**
    * @dev Calculate amount of tokens for a given wei amount. Apply special bonuses depending on
    * @param weiAmount Amount of wei for token purchase.
-   * the time of the purchase. If purchase is outside of every token sale window then revert.
-   * @return number of tokens per wei amount with possible token bonus
+   * @param bonusPercent Percentage of bonus tokens.
+   * @return Number of tokens with possible bonus.
    */
-  function getTokenAmount(uint256 weiAmount)
+  function getTokenAmount(uint256 weiAmount, uint256 bonusPercent)
     internal
     view
     returns(uint256)
   {
     uint256 tokens = weiAmount.mul(rate);
-    uint256 bonus = 0;
+    uint256 bonusTokens;
```

Comment:
Maybe this could be defined inside if statement since it is not used otherwise.

Reply:
7191544